packages/components/src/components/input/input.tsx

```diff
@@ -28,11 +28,13 @@ type TInputProps = {
     name?: string;
     onBlur?: (e: React.FocusEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
     onChange?: (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+    onKeyUp?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+    onKeyDown?: (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
     onFocus?: (e: React.FocusEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
     onPaste?: (e: React.ClipboardEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
     placeholder?: string;
     required?: boolean;
-    trailing_icon?: React.ReactElement;
+    trailing_icon?: React.ReactElement | null;
```

Comment:
Allowing `null` here because of this https://github.com/binary-com/deriv-app/pull/7385/files#diff-f9654ab9677879818ad0d46350d6373f45633d516aa831d8e74ecca4b9222cc6L48-R70

This prop could be alternatively defined as `ReactNode`, but I thought it's better to allow one more possible option (`null`), rather than lots of them (`number`, `string`, virtually anything renderable, as `ReactNode` is exactly for that).